rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,507 @@
+
+### **Acceptance Criteria**
+
+Except where listed ("responsible team"), the starfish and OME teams
+will do the majority of project management and execution for the
+following deliverables. They will coordinate and collaborate with DCP
+partners ("contact person") to validate design correctness and minimize
+code changes and their impact.
+
+| Component/Deliverable                                 	| Responsible Team/Service 	| No earlier than 	| Contact Person	|
+|-------------------------------------------------------	|--------------------------	|-----------------	|---------------	|
+| HCA Metadata Draft                                    	| EBI                      	| Complete        	| Z. Perova     	|
+| Test Datasets Available                               	| starfish                 	| Complete        	| A. Carr       	|
+| Submission Datasets Available                         	| starfish                 	| Q2 2019         	| A. Carr       	|
+| Analysis & Adapter Pipelines                          	| Broad                    	| Q2 2019         	| A. Carr       	|
+| Lira subscription                                     	| Broad                    	| ?             	| S. Ehsan      	|
+| Data Processing Pipelines Tests                        	| Broad                    	| Q2 2019         	| G. Grant      	|
+| Specification for input format                        	| starfish                 	| Complete        	| J. Moore      	|
+| Specification for output format                       	| starfish                 	| Q1 2019         	| A. Carr       	|
+| Input validation tool                                 	| starfish, OME            	| Q3 2019         	| J. Moore      	|
+| Output validation tool                                	| starfish, OME            	| Q2 2019         	| A. Carr       	|
+| Ingest support for the nested arrays of channels      	| EBI                      	| ?               	| Z. Perova           |
+| [Upload and download of subdirectories][b]             	| ?                        	| ?               	| H. Schmidt           |
+| DSS spatial data bundle (or bundless) specification   	| UCSC                     	| ?               	| B. Hannafious 	|
+| DSS scale testing                                     	| UCSC / DCP               	| ?             	| B. Hannafious 	|
+| Data Browser facets for spatial data                  	| UCSC                     	| ?             	| H. Schmidt    	|
+| Matrix service integration                            	| starfish                 	| ?             	| M. Kinsella   	|
+| Upload Bundle for 1M+ files                           	| ?                        	| ?               	| P. Shah       	|
+| Image file download latency testing                   	| ?                        	| ?               	| B. Hannafious             	|
+
+### **Unresolved Questions**
+
+  - Who are the intended beta users for imaging support and what
```

Comment:
How do you plan to resolve this question?

Reply:
I removed this question, but the short answer is that it's implicit in the outlined features: The lack of visualization support means we're targeting users who are comfortable interacting with count matrices or raw image data. These is our _user with a keyboard_.